pkg/api/types.go

```diff
@@ -2152,6 +2152,8 @@ type SecretList struct {
 	Items []Secret `json:"items"`
 }
 
+// +genclient=true
```

Comment:
almost, there's a bug in the client generation that meant I had to write the `ConfigMapExpansion` declaration myself.